### PR TITLE
Moved imports for Python 3 (unichr, reduce, xrange)

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: translate/lang/ngram.py [hunk 1]
```diff
@@ -131,7 +132,7 @@ def classify(self, text):
         ngram = _NGram(text)
         r = 'guess'
 
-        min = sys.maxint
+        min = six.MAXSIZE
```
Comment (unho) [inline translate/lang/ngram.py]: Note that `six.MAXSIZE` is not a replacement for `sys.maxint` https://pythonhosted.org/six/#six.MAXSIZE After reading the code where `min` is used I suppose this works fine.

**Comment by unho** [comment]
Will merge after rebase.

File: translate/lang/ngram.py (hunk 1 above)
Comment (claudep) [inline translate/lang/ngram.py]: I suppose so, too.

**Comment by claudep** [comment]
Rebased.

**Comment by unho** [comment]
Merged. Thanks!

**Pull request merged**